### PR TITLE
Change categories to checkboxes

### PR DESCRIPTION
**Pull request by Karyum**
#153

### CODE REVIEW

File: routes/home.js [hunk 1]
```diff
@@ -24,9 +24,20 @@ module.exports = async (req, res) => {
       );
     }
 
+    const categories = Object.keys(
+      events
+        .reduce((acc, { categories }) => [...acc, ...categories], [])
+        .reduce((acc, category) => {
```
Comment (finnhodgkin) [inline routes/home.js]: You can reduce this to a single reduce using `set`, but I have a feeling babel will make it very long so it's up to you:

```js
events.reduce((acc, { categories }) => [...new Set([...acc, ...categories])], [])
```

File: views/helpers/upperCase.js [hunk 2]
```diff
@@ -0,0 +1 @@
+module.exports = word => word[0].toUpperCase() + word.slice(1);
```
Comment (finnhodgkin) [inline views/helpers/upperCase.js]: This helper could be replaced with CSS if you wanted:
```css
text-transform: lowercase;
```

File: views/partials/header.hbs [hunk 3]
```diff
@@ -15,18 +15,10 @@
...
-      <option value="sport">Sport</option>
-      <option value="competition">Competition</option>
-      <option value="launch">Launch</option>
-      <option value="party">Party</option>
-      <option value="culture">Culture</option>
-      <option value="miscellaneous">Miscellaneous</option>
-    </select>
+    {{#each categories}}
+    <label>{{upperCase this}}
+      <input type="checkbox" name="" value="{{this}}">
```
Comment (finnhodgkin) [inline views/partials/header.hbs]: Add the value to the name or just remove the attribute.

File: views/partials/header.hbs [hunk 4]
```diff
@@ -15,18 +15,10 @@
...
-      <option value="conference">Conference</option>
-      <option value="sport">Sport</option>
-      <option value="competition">Competition</option>
-      <option value="launch">Launch</option>
-      <option value="party">Party</option>
-      <option value="culture">Culture</option>
-      <option value="miscellaneous">Miscellaneous</option>
-    </select>
+    {{#each categories}}
+    <label>{{upperCase this}}
```
Comment (finnhodgkin) [inline views/partials/header.hbs]: On second thoughts the designs are actually all lower case...

https://user-images.githubusercontent.com/22300773/33836275-a0ae88ec-de91-11e7-85ce-8da26a515a7c.png

File: routes/home.js (hunk 1 above)
Comment (finnhodgkin) [reply routes/home.js]: So you decided on the old method..?

Comment (Karyum) [reply routes/home.js]: yeah

**Pull request merged**